### DISCUSSION TITLE
Language getShopReceipts

### DISCUSSION THREAD

**Discussion by scuculeac-plenty** [etsy/open-api#850] [Q&A] [answered]
On v2 at some point we could send language on the request, but in v3 we use getShopReceipts and we get transactions on English even if we have shop default language as de.

When we try in post man https://openapi.etsy.com/v2/listings/:listing_id i get data in english ("language": "en-US",) but when we get transactions from getShopReceipts we get it in DE., in other case get listing details are "language": "MACHINE_en", which seems to be related to settings > options > Automatic Listing Translation but in getShopReceipts we get them in other language.

How can we get getShopReceipts in DE, based on which setting?

Thank you
Best Regards
Stefan

**Comment by scuculeac-plenty** [comment]
Any news? it's a big problem for us and for the customers

**Comment by m-siebert** [comment]
I'm also interested in a solution here!

**Comment by scuculeac-plenty** [comment, marked as answer]
language=de as a query parameter to force-get the de translation -> this works if anyone needed resolved
